LaCE-guide.md

```diff
@@ -0,0 +1,63 @@
+  - Diagnostic Tests: Add skills that will be used to generate the diagnostic test that determines whether to recommend this topic to new learners. For a skill to qualify for this, it must have at least 3 questions and be assigned to that topic.
+  - Canonical stories: In this section, the stories belonging to the topic are listed.
+
+## 2. Creating a Skill
+  > **_NOTE:_** For a skill to be visible on the practice tab of a topic, it must have atleast 10 questions associated with it.
+- A skill can be directly created from the topic editor by clicking the `ADD SKILL` button under `Subtopics` section of the topic editor. But in this case, the skill will be automatically assigned to that topic.
+- A skill can also be created from the Topic and Skill Dashboard as well but it will not be assigned to any topic.
+- On clicking the `ADD SKILL` option a modal will pop up with necessary details of the skill.
+- Once this is done, the skill is created and you will be redirected to the skill editor page
+- The main skill editor section has the following part:
+  - Details
+  - Worked example
+  - Misconceptions: Common errors that users may have and their explanations.
+  - Pre-requisite Skills: Skills that are to be completed before attempting the current skill.
+- The questions section lists the questions in the skill:
+  - To add a new question, select the `ADD QUESTION` option
+  - In the question editor, there are 5 sections:
+    - Difficulty
+    - Problem
+    - Interaction
+    - Answers ans Responses
+    - Hints
+  - Once all these sections are filled, the question can be saved.
+
+## 3. Generating data on localhost
+- To generate data, we must first navigate to the admin page. We will automatically be redirected to the `ACTIVITIES`.
+- This tab will have several sections:
+  - Reload a single exploration: This section has preloaded explorations. These can be loaded onto your local server by clicking the `Reload` button for each of them.
+  - Generate dummy explorations: Along with the option to generate dummy explorations (with no content), we can also publish a certain number of them.
+  - Reload a single collection: This section has preloaded collections.
+  - Load dummy new structures data: This section provides a combination of topics, skills, stories and exploration.
+  - Generate dummy skill with questions
+  - Generate a dummy math classroom
+
+## 4. Creating a Classroom
+
+- We must first navigate to the `CONFIG` tab in the admin page. 
+- In this tab, there is a section `The details for each classroom page.` in which will find a defult math classroom that is filled out. 
+- It has details about the classroom:
+  - Name
+  - URL Fragment
+  - topic list intro
+  - topci ids: The topic ids that should included in the classroom are listed here. We can add to this by clicking on the `Add element` option.
+- More classrooms can be generated by clicking on the `Add element` option beneath this section.
```

Comment:
Don't they need to save the new config at the end? Explain what they should click on and what message they should expect to see if successful. Explain also where to find the resulting classroom (which URL).